core/internal/filetransfer/file_transfer_manager.go

```diff
@@ -101,11 +101,11 @@ func (fm *fileTransferManager) AddTask(task *Task) {
 		if task.Err != nil {
 			fm.logger.CaptureError(
 				fmt.Errorf(
-					"filetransfer: uploader: error uploading path=%s url=%s: %v",
-					task.Path,
-					task.Url,
```

Comment:
this was shoveled to the err msg bc the max tag value length is 200 characters in sentry, so will likely be truncated. could you please verify?

Reply:
Oh, I didn't realize that! The URL will get truncated if that's the max length. The path is less likely to be truncated.

They recommend breaking it up into multiple pieces as a workaround: https://docs.sentry.io/platforms/python/tracing/troubleshooting/#control-data-truncation (funnily, they don't have any recommendation if you change the language to Go)

I'll just leave the URL in the message.